test/ts/unit/collateralized_contract.ts

```diff
@@ -35,38 +35,45 @@ contract("CollateralizedContract (Unit Tests)", async (ACCOUNTS) => {
     let mockToken: MockERC20TokenContract;
     let mockRegistry: MockDebtRegistryContract;
 
-    const CONTRACT_OWNER = ACCOUNTS[0];
+    const CUSTODIAN = ACCOUNTS[0];
     const COLLATERALIZER = ACCOUNTS[1];
     const BENEFICIARY = ACCOUNTS[2];
     const ATTACKER = ACCOUNTS[3];
 
-    const TX_DEFAULTS = { from: CONTRACT_OWNER, gas: 4000000 };
+    const TX_DEFAULTS = { from: CUSTODIAN, gas: 4000000 };
```

Comment:
This is misleading title change -- the contract owner (i.e. the person who _deployed_ the contract) is at on point in time the custodian of the collateral.  Rather, the `CollateralizedContract` (deployed by `ACCOUNTS[0]`) is the custodian.  Please fix